package.json

```diff
@@ -34,12 +34,13 @@
   },
   "homepage": "https://github.com/reduct/registry",
   "devDependencies": {
-    "@reduct/build-tools": "^1.4.5"
+    "@reduct/build-tools": "^1.5.2",
+    "@reduct/logger": "^1.1.0"
   },
   "reduct": {
     "sourceFolder": "Src/",
     "distributionFolder": "Dist/",
     "entryFile": "Registry.js",
-    "globalPackageName": "registry"
+    "globalPackageName": "reductRegistry"
```

Comment:
Hey @akoenig - This is due to the fact that the UMD-Wrapper itself is now handled by browserify, since I couldn't export a clean browserify build through our UMD-Wrapper.

A clean browserify bundle will always return the following function when the `standalone`/`umd` option was *not* enabled:

`function s(o,u)`  [link](https://github.com/reduct/registry/blob/task/moveToCleanES6ModuleStructure/Dist/Registry.js#L37)

This function always expects arguments which are internals of the bundle itself. Thus I had to enable the `standalone` option, and this option converts, as it is with any umd wrapper option out there, any dots in our global namespace into a camelCase name.

So basically the reduct object in the global namespace is now deprecated. and should only be initiated if necessary.